desc/grid.py

```diff
@@ -48,6 +48,21 @@ class _Grid(IOAble, ABC):
         "_fft_toroidal",
     ]
 
+    _static_attrs = [
+        "_endpoint",
+        "_can_fft2",
+        "_coordinates",
+        "_fft_poloidal",
+        "_fft_toroidal",
+        "_is_meshgrid",
+        "_L",
+        "_M",
+        "_N",
+        "_NFP",
+        "_node_pattern",
+        "_sym",
```

Comment:
why would we not want `_period` also here?

Reply:
I don't think we ever do any conditionals based on `period` so there's no need for it to be static